collections/editor/observationsubmit.php

```diff
@@ -76,14 +78,14 @@
 	$displayLeftMenu = (isset($collections_editor_observationsubmitMenu)?$collections_editor_observationsubmitMenu:false);
 	include($SERVER_ROOT.'/includes/header.php');
 	echo '<div class="navpath">';
-	echo '<a href="../../index.php">Home</a> &gt;&gt; ';
+	echo '<a href="../../index.php">' . (isset($LANG['HOME'])?$LANG['HOME']:'Home') . '</a> &gt;&gt; ';
 	if(isset($collections_editor_observationsubmitCrumbs)){
 		echo $collections_editor_observationsubmitCrumbs;
 	}
 	else{
-		echo '<a href="../../profile/viewprofile.php?tabindex=1">Personal Management</a> &gt;&gt; ';
+		echo '<a href="../../profile/viewprofile.php?tabindex=1">' . (isset($LANG['PERS_MNGT'])?$LANG['PERS_MNGT']:'Personal Management') . '</a> &gt;&gt; ';
```

Comment:
Add spaces to the ternary operator syntax:
`foo ? bar : baz` rather than `foo?bar:baz`.